eng/Versions.props

```diff
@@ -261,6 +261,7 @@
     <XunitExtensibilityExecutionVersion>$(XunitVersion)</XunitExtensibilityExecutionVersion>
     <XUnitRunnerVisualStudioVersion>2.4.3</XUnitRunnerVisualStudioVersion>
     <MicrosoftDataSqlClientVersion>1.0.19249.1</MicrosoftDataSqlClientVersion>
+    <MicrosoftAspNetCoreAppVersion>6.0.0-preview.3.21167.1</MicrosoftAspNetCoreAppVersion>
```

Comment:
There's a `SharedFxVersion` variable that's configured at the repo level. (I think). You should use that

Reply:
See https://github.com/dotnet/aspnetcore/blob/main/Directory.Build.targets#L47-L48. The variable is defined in every project that imports the Directory.Build.targets file. Note some projects are isolated i.e. don't import that file, some are test assets we want to be as realistic / normal as possible while other projects simply aren't .NET SDK projects.